README.md

```diff
@@ -17,10 +17,42 @@ There you'll find documentation, examples, announcements and more.
 
 ### Configuration
 
-Feedjira configuration options can be set by using a block
+#### Stripping whitespace from XML
+
+Feedjira can be configured to strip all whitespace but defaults to lstrip only:
 
 ```ruby
 Feedjira.configure do |config|
   config.strip_whitespace = true
 end
 ```
+
+#### Follow redirect limit
+
+For fetching feeds, the follow redirect limit defaults to 3 but can be set:
+
+```ruby
+Feedjira.configure do |config|
+  config.follow_redirect_limit = 5
+end
+```
+
+#### Request timeout
```

Comment:
I can't agree with that. When someone uses a library he should be able to read the documentation. The README shall be a clear and simple document - not blown up with a bunch of unnecessary lines of code.

It should be possible to set configuration without a block (that's what a user will do, if he isn't interested in setting multiple options). So you could replace 

```ruby
Feedjira.configure do |config|
   config.follow_redirect_limit = 5
end
```

with

```ruby
  Feedjira.follow_redirect_limit = 5
```